roles/engine-api/files/wazo-auth.yml

```diff
@@ -1,5 +1,2 @@
-uuid: undefined
-enabled_backend_plugins:
-  ldap_user: true
 service_discovery:
   enabled: false
```

Comment:
This file can be completely removed: we want service discovery to be enabled!

Reply:
ok removing the file but I have no idea what it does (that was a change from Alex ;-) )